File: app/src/main/java/com/lin/lnpull/ListLoading.java
package com.lin.lnpull;

import android.view.View;
import android.widget.LinearLayout.LayoutParams;
/**
 * author : leo
 * date   : 2018/10/2814:13
 */
public interface ListLoading {
	public View  getLoadingView();
	public View  getErrorView();
	public void startLoading();
	public void finishLoading();
	public void finishLoadingNoAnimation();
	public void  showErrorView();
	public void showEmptyView();
	public void addBottomView(View view);
	public void addBottomView(View view, LayoutParams params);
}

File: app/src/main/java/com/lin/pull/ListLoading.kt
package com.lin.pull

import android.view.View
import android.widget.LinearLayout.LayoutParams
/**
 * author : leo
 * date   : 2018/10/2812:07
 */
interface ListLoading {
    fun startLoading()
    fun finishLoading()
    fun finishLoadingNoAnimation()
    fun showErrorView()
    fun showEmptyView()
    fun addBottomView(view: View)
    fun addBottomView(view: View, params: LayoutParams)
}
